tests/unit/test-class-wc-payment-gateway-wcpay-subscriptions.php

```diff
@@ -512,6 +512,34 @@ public function test_render_custom_payment_meta_input_invalid_value() {
 		$this->wcpay_gateway->render_custom_payment_meta_input( $subscription, 'field_id', 'invalid_value' );
 	}
 
+	public function test_render_custom_payment_meta_input_multiple_tokens() {
+		$subscription = WC_Helper_Order::create_order( self::USER_ID );
+		$tokens       = [
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_1', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_2', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_3', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_4', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_5', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_6', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_7', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_8', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_9', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_10', self::USER_ID ),
+			WC_Helper_Token::create_token( self::PAYMENT_METHOD_ID . '_11', self::USER_ID ),
```

Comment:
@nagpai , can we keep only 3 of them here - it shall be enough for test and easier to read the test case.

Reply:
An [earlier PR](https://github.com/Automattic/woocommerce-payments/pull/1097) cited in our [related issue](https://github.com/Automattic/woocommerce-payments/issues/3924#issuecomment-1102378293), fixed an error when there were more than 10 cards. There was no accompanying test to that. Hence thought of testing this with 11 to be safe. 

Let me know if it makes sense to keep it at 11, or reduce to three or lesser number.

Reply:
Gotcha! Then, let's stick to those 11 tokens.